src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,113 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
 
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
+import { findMovie } from '../../api/api';
+
+export const FindMovie = ({ addToList }) => {
+  const [query, setQuery] = useState('');
+  const [newMovie, setNewMovie] = useState(null);
+  const [isHiddenWarning, setIsHiddenWarning] = useState(true);
+
+  const handleQuery = async(title) => {
+    const movie = await findMovie(title);
+
+    if (movie.Response === 'True') {
+      setNewMovie({
+        title: movie.Title,
+        description: movie.Plot,
+        imgUrl: movie.Poster,
+        imdbUrl: `https://www.imdb.com/title/${movie.imdbID}/`,
+        imdbId: movie.imdbID,
+      });
+      setIsHiddenWarning(true);
+    } else {
+      setIsHiddenWarning(false);
+    }
+  };
+
+  const handleAdding = () => {
+    if (newMovie) {
+      addToList(newMovie);
+      setQuery('');
+    } else {
+      setIsHiddenWarning(false);
```

Comment:
consider of cleaning newMovie after adding it to the list

Reply:
fixed